plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/jsonata/jsonata.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
+import jsonata from 'jsonata';
+
+export function createJSONataAction() {
+  return createTemplateAction<{
+    data: any;
+    expression: string;
+  }>({
+    id: 'roadiehq:utils:jsonata',
+    description:
+      'Allows performing jsonata opterations and transformations on objects',
```

Comment:
What's the difference between this action and the :json action? They have the same description.

Reply:
This one, takes input data object (i.e. not from a file) and produces a step output (not written to a file).

The json:transform task, reads from a json file and optionally writes back to it.

Ill update the description.